api/nodes/views.py

```diff
@@ -40,7 +40,7 @@ def get_default_odm_query(self):
         return (
             Q('is_public', 'eq', True) &
             Q('is_deleted', 'ne', True) &
-            Q('is_dashboard', 'ne', True)
```

Comment:
In the future, we might add a "dark" /dashboard/ endpoint which we will use to create the project organizer, but I don't think it makes sense to return folders for /nodes/